pdp/expr_try.go

```diff
@@ -0,0 +1,54 @@
+package pdp
+
+type functionTry struct {
+	args []Expression
+}
+
+func makeFunctionTry(args []Expression) Expression {
+	return functionTry{
+		args: args,
+	}
+}
+
+func (f functionTry) GetResultType() Type {
+	if len(f.args) > 0 {
+		return f.args[0].GetResultType()
+	}
+
+	return TypeUndefined
+}
+
+func (f functionTry) describe() string {
+	return "try"
+}
+
+func (f functionTry) Calculate(ctx *Context) (AttributeValue, error) {
+	var (
+		v   AttributeValue
+		err error
+	)
+
+	for _, arg := range f.args {
+		v, err = arg.Calculate(ctx)
```

Comment:
would be nice to check the value and continue if empty

Reply:
Not every type has "empty" value.

Reply:
ok, looks good